packages/core/src/dialects/postgres/query-interface.js

```diff
@@ -260,4 +260,28 @@ export class PostgresQueryInterface extends AbstractQueryInterface {
 
     await Promise.all(promises);
   }
+
+  /**
+ * Get all tables in current database
+ *
+ * @param {object}    [options] Query options
+ * @param {boolean}   [options.raw=true] Run query in raw mode
+ * @param {QueryType} [options.type=QueryType.SHOWTABLE] query type
+ * @param {string} [options.schema]
+ *
+ * @returns {Promise<Array>}
+ * @override
+ */
+  async showAllTables(options) {
+    options = {
+      ...options,
+      raw: true,
+      type: QueryTypes.SHOWTABLES,
+    };
+
+    const showTablesSql = this.queryGenerator.showTablesQuery(options.schema);
```

Comment:
this shouldn't be put at the postgres level, it should have the same behavior across all dialects so it should go in AbstractQueryInterface

Reply:
Updated abstract showAllTables instead.